python/cuml/pytest_benchmarks/test_ensemble.py

```diff
@@ -0,0 +1,226 @@
+    setup_cpu_func=None,
+    setup_cuml_func=None)
+
+
+@pytest.mark.integration
+@pytest.mark.skipif(not has_pytest_benchmark(),
+                    reason='pytest-benchmark missing')
+@pytest.mark.parametrize('n_samples', [unit_param(500), stress_param(50000)])
+@pytest.mark.parametrize('n_features', [5, 50])
+@pytest.mark.parametrize('algotype', ['gpu', 'cpu'])
+def test_ensemble_stacking_classifier(gpubenchmark, n_samples,
+                                      n_features, algotype):
+    _ensemble_benchmark_algo(gpubenchmark, stackingclassifier,
+                             dataset_func=make_classification,
+                             n_samples=n_samples, n_features=n_features,
+                             data_kwargs={'n_redundant': 0,
+                                          'class_sep': 0.75,
+                                          'n_classes': 2},
+                             algo_type=algotype)
+
+
+#
+# Bagging Regressor
+#
+baggingregressor = AlgorithmPair(
+    cpu_class=BaggingRegressor,
+    cuml_class=BaggingRegressor,
+    cuml_args={'base_estimator':
+               cuml.neighbors.KNeighborsRegressor()},
+    cpu_args={'base_estimator':
+              KNeighborsRegressor(n_jobs=-1),
+              'verbose': 2,
+              'n_jobs': -1},
+    shared_args={'n_estimators': 20,
+                 'random_state': 12},
+    name="BaggingRegressor",
+    accepts_labels=False,
+    accuracy_function=None,
+    bench_func=fit_and_predict,
+    setup_cpu_func=None,
+    setup_cuml_func=None)
+
+
+@pytest.mark.integration
+@pytest.mark.skipif(not has_pytest_benchmark(),
+                    reason='pytest-benchmark missing')
+@pytest.mark.parametrize('n_samples', [unit_param(500), stress_param(250000)])
+@pytest.mark.parametrize('n_features', [5, 20])
+@pytest.mark.parametrize('algotype', ['gpu', 'cpu'])
+def test_ensemble_bagging_regressor(gpubenchmark, n_samples,
+                                    n_features, algotype):
+    _ensemble_benchmark_algo(gpubenchmark, baggingregressor,
+                             dataset_func=make_regression,
+                             n_samples=n_samples, n_features=n_features,
+                             data_kwargs={'random_state': 12, 'noise': 200},
+                             algo_type=algotype)
+
+
+#
+# Boosted Regression
+#
+
+boostedregressor = AlgorithmPair(
+    cpu_class=AdaBoostRegressor,
+    cuml_class=AdaBoostRegressor,
+    cuml_args={'base_estimator': cuml.svm.SVR()},
+    cpu_args={'base_estimator': SVR()},
+    shared_args={'n_estimators': 10,
+                 'random_state': 12},
+    name="BoostedRegressor",
+    accepts_labels=False,
+    accuracy_function=None,
+    bench_func=fit_and_score,
+    setup_cpu_func=None,
+    setup_cuml_func=None)
+
+
+@pytest.mark.integration
+@pytest.mark.skipif(not has_pytest_benchmark(),
+                    reason='pytest-benchmark missing')
+@pytest.mark.parametrize('n_samples', [unit_param(50), stress_param(20000)])
```

Comment:
Similar comment as above - 50 rows seems very small?

Reply:
same as above.  Happy to revise the unit_params to a value that will still run quickly for the cpu implementation but will remain stable.